packages/web-components/src/switch/switch.stories.ts

```diff
@@ -1,106 +1,140 @@
-import { html } from '@microsoft/fast-element';
-import type { Args, Meta } from '@storybook/html';
-import { renderComponent } from '../helpers.stories.js';
+import { html, repeat } from '@microsoft/fast-element';
+import { uniqueId } from '@microsoft/fast-web-utilities';
```

Comment:
just a note that this lib is getting deprecated soon and we should disconnect the dependency (not in this PR but generally)